src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -527,8 +527,12 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig,
     for {
       paramMap <- processListClustersParameters(params)
       clusterList <- dbRef.inTransaction { da => da.clusterQuery.listByLabels(paramMap._1, paramMap._2, googleProjectOpt) }
-      visibleClusters <- authProvider.filterUserVisibleClusters(userInfo, clusterList.map(c => (c.googleProject, c.clusterName)).toList)
+      samVisibleClusters <- authProvider.filterUserVisibleClusters(userInfo, clusterList.map(c => (c.googleProject, c.clusterName)).toList)
+      // Making the assumption that users will always be able to access clusters that they create
```

Comment:
i think this is a safe assumption to make. per [sam's config](https://github.com/broadinstitute/sam/blob/develop/src/main/resources/reference.conf#L286-L317), we don't even give people the permission to add/remove others from a cluster so once somebody creates a cluster they should never be removed.

at the very least this isn't driving access so even if the above weren't true, it wouldn't let anybody access data/clusters they shouldn't be able to see or didn't know about already.